resample/bootstrap.py

```diff
@@ -141,8 +141,9 @@ def confidence_interval(
     cl : float, default : 0.95
         Confidence level. Asymptotically, this is the probability that the interval
         contains the true value.
-    ci_method : str, {'percentile', 'student', 'bca'}, optional
-        Confidence interval method. Default is 'percentile'.
+    ci_method : str, {'percentile', 'bca'}, optional
+        Confidence interval method. Default is 'percentile'. The 'bca' method should be
+        more accurate (2nd order accurate) for the same number of function evaluations.
```

Comment:
Is there any benefit to the percentile method we can point out (slightly easier interpretation / computation)? Else it sounds like one method completely dominates the other and the other should never be used.

(Might as well make bca the default here and mark the issue as being closed by this PR.)

Reply:
I think using bca as default makes a lot of sense. As far as I understand from the book, it should be always better, but I think we should offer the simple percentile method as an option as well, because it is the simple straight-forward estimate. There is no way that the percentile method can be wrong, so it is a good fallback. It is also good to check the bca method.

Another reason to keep the percentile method is that the bca method uses a jackknife estimate in addition to the bootstrap estimate. If your sample is very large, n > 10000, the jackknife will become the bottleneck. The percentile method does not need a jackknife estimate and just scales in computational time with the number of bootstrap replicas. So there are definitely possible applications where one would chose the percentile method.

Reply:
I will change to bca as the default then.